src/PhpBrew/Command/EachCommand.php

```diff
@@ -2,72 +2,17 @@
 
 namespace PhpBrew\Command;
 
-use PhpBrew\Config;
-use PhpBrew\PhpBrew;
 use Exception;
 
 class EachCommand extends \CLIFramework\Command
```

Comment:
You may extends `VirtualCommand` directly, then `execute` can be removed entirely.